<?xml version="1.0" ?>
<launch>

<node name="capturedata" pkg="capturedata" type="capturedata_flightroom" output="screen" >
    <!-- <remap from="/tf" to="/allegroHand_0/tf" /> -->
    <!-- <remap from="/allegroHand_0/tf_static" to="/tf_static" /> -->

 </node>
 
<node name="capturedata_keyboard" pkg="capturedata" type="capturedata_key" respawn="false" output="screen"> </node> 



<!-- rosrun dynamic_reconfigure dynparam set /camera/rgb_camera 
"{'white_balance':3600, 'enable_auto_exposure':False, 'exposure':40, 'enable_auto_white_balance':False, 'auto_exposure_priority':False}" -->
<!-- Below is same as upper command. Manually found the exposure / WB values.  -->

<!-- http://wiki.ros.org/hokuyo_node/Tutorials/UsingDynparamToChangeHokuyoLaserParameters -->
<!-- http://wiki.ros.org/rqt_reconfigure -->
  <node name="$(anon dynparam)" pkg="dynamic_reconfigure" type="dynparam" args="set_from_parameters /camera/rgb_camera">


    <param name="white_balance" type="double" value="3600" />

    <param name="enable_auto_exposure" type="bool" value="False" />
    <param name="enable_auto_white_balance" type="bool" value="False" />
    <param name="auto_exposure_priority" type="bool" value="False" />
    <param name="exposure" type="int" value="100" />


  </node>


</launch>
